docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -136,10 +136,10 @@ connector.
       - Description
       - Default
     * - ``delta.domain-compaction-threshold``
-      - Sets the number of transactions to act as threshold. Once reached the
-        connector initiates compaction of the underlying files and the delta
-        files. A higher compaction threshold means reading less data from the
-        underlying data source, but a higher memory and network consumption.
+      - Sets the number of transactions to act as a threshold. After reaching
+        the threshold, the connector initiates compacting a large IN or OR
+        clause into a min-max range predicate for pushdown into an ORC or
+        Parquet reader.
```

Comment:
We must be less specific about this.
The compaction happens at _some_ stage of query processing and affects _some_ further phases of query processing (including file-level pruning if applicable).